frontend/src/components/DiscoveryHook/EnhancedHero.tsx

```
@@ -99,21 +83,10 @@ const EnhancedHero = ({
-            <div className="flex items-center gap-2 text-blue-300">
-              <Shield size={16} />
-              <span>Risk-free guarantee</span>
-            </div>
-            <div className="flex items-center gap-2 text-purple-300">
-              <Sparkles size={16} />
-              <span>2-minute setup</span>
-            </div>
-          </div>
+          {/* Short genuine tagline */}
+          <p className="text-[#E6F6FF] opacity-90 font-manrope mb-6 max-w-3xl mx-auto">
+            We're polishing the experience — sign up below to be among the first to try it.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)